22.2.3+ (#1)
friedcucumber/maui-cards-with-custom-content: README.md

<!-- default badges list -->
<!-- default badges end -->
## .NET MAUI Controls - Cards with Custom Content

> This example shows how to display a [SimpleButton](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton) and [ChartView](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.ChartView) controls within cards. You can define any required actions for these cards.  

Reviewer: > This example **shows** how to display **a** [SimpleButton](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton) and [ChartView](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.ChartView) controls **within** cards. You can define any required actions for these cards.

Indefinite Article Usage Case:

We list controls in plural form, therefore, we need "zero article" in this case -- `SimpleButton` and `ChartView` controls. Please refer to the new guideline on articles for details: http://nk-zero-doc-issues.tw.devexpress.devx/language-guidelines/articles/

Within VS In - I recommend using a more abstract preposition because we deal more in logical relationship than in geometry.

Shows vs Demonstrates/Illustrates. While one definitely can say "shows" in this context, the purpose of an example is to demonstrate or illustrate some concept or scenario. This is a minor issue but please pay a bit more attention to such things in the future:

However, more importantly, this introduction should fall in line with the corresponding documentation topic:

```suggestion
This example demonstrates how to turn [SimpleButton](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton) controls into clickable cards with custom content. For example, the topmost card displays a [ChartView](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.ChartView). 
```

Please refer to the following review on the GitLab platform for details:

https://git.internal.devexpress.devx/Documentation/maui/-/merge_requests/760

<img src="https://user-images.githubusercontent.com/12169834/228201579-826dea6c-4af0-4bdc-ae86-4873cd115fd6.png" width="30%"/>

> Used controls and their properties:

Reviewer: > Used controls and their properties:

"Used controls" sounds a bit awkward and strange. I wouldn't have used "used" because it sounds something like "second-hand controls", which is not the thing we'd like to convey.

My current take on this introduction is the following:

```suggestion
Included controls and their properties:
```

* [Label](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/label?view=net-maui-7.0): [Text](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.label.text?view=net-maui-7.0)

* [SimpleButton](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton): [Content](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton.Content), [Clicked](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton.Clicked)

> If a control within the [SimpleButton](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton) intercepts a click, set the control's `InputTransparent` property to `true`:

Reviewer: > If a control within **the** [SimpleButton](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton) intercepts a click, set the control's `InputTransparent` property to `true`:

Do we speak of a specific SimpleButton control instance in this case? From my point of view, we speak of a general case -- when any control in a SimpleButton intercepts a click, set the control's InputTransparent property to true.

This paragraph does not explain why should we perform the described actions. What do we accomplish here by doing so? Right now, it's unclear.

Author: ```suggestion
If a control within a [SimpleButton](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton) intercepts a click, set the control's `InputTransparent` property to `true`. In this case, a control within the [SimpleButton](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton) handles clicks:
```

Reviewer: OK. This sounds better indeed. However, I've got the following questions here:

1. Is it that important to emphasize the fact that the `InputTransparent` property is inherited?
2. From what I gather, the mentioned control has the `SimpleButton` property. Just to satisfy my curiosity -- does this control indeed has a dedicated property for associating it with a simple button? I just need to make sure that we haven't made an accidental mistake here.
3. Within VS In -- As I mentioned earlier, "within" emphasizes geometrical relationship -- that is, a control placed within a simple button's boundaries, while "In" is more general and, therefore, more suitable for explaining logical relationship.

Author: I've updated my suggestion and fix some issues.
